app/classes/document_query.rb

```diff
@@ -65,6 +71,32 @@ def aggregation_hash(facet_field)
     }
   end
 
+  def date_aggregation_hash(date_facet_field)
```

Comment:
Is this method used anywhere outside of this class? If not, it can be made private.

Reply:
@ddzz Moved this, as well as the two methods it was based on (which are also only called from this class -- they're probably more) so as not to divorce it from this helpful context.